package.json

```diff
@@ -62,6 +62,7 @@
     "node-sass": "^4.14.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-markdown": "^3.6.0",
```

Comment:
I suspect you are matching the version used in Interviewer here, but I think we should go with the newer version `5.0.3`. It does change the API slightly (no `source` prop anymore -> use `children`). Reason for this is that there are a number of bug fixes and improvements, plus it supports React 17, so it will be one less thing to fix when we address that.

Reply:
Good point! Updated versions here.